tests/python/driver/tvmc/test_frontends.py

```diff
@@ -115,26 +115,38 @@ def test_load_model__tflite(tflite_mobilenet_v1_1_quant):
     assert "_param_1" in params.keys()
 
 
-def test_load_model__keras(keras_resnet50):
+def verify_load_model__keras(model, **kwargs):
     # some CI environments wont offer TensorFlow/Keras, so skip in case it is not present
     pytest.importorskip("tensorflow")
 
-    mod, params = tvmc.frontends.load_model(keras_resnet50)
+    mod, params = tvmc.frontends.load_model(model)
     assert type(mod) is IRModule
     assert type(params) is dict
     ## check whether one known value is part of the params dict
     assert "_param_1" in params.keys()
 
 
-def test_load_model__onnx(onnx_resnet50):
-    # some CI environments wont offer onnx, so skip in case it is not present
-    pytest.importorskip("onnx")
+def test_load_model__keras(keras_resnet50):
```

Comment:
A more Pythonic way to do this is to have a single test case (`test_load_model__keras`) and use `@pytest.mark.parametrize(...)` to specify the different test cases. In this case you will want to pass the full parameters to load_model as a single `kwargs` dict, so it would be something like:

```
@pytest.mark.parametrize("load_model_kwargs", [ {"path": keras_resnet50}, {"path": keras_resnet50, "layout": "NCHW"}])
def test_load_model__keras(load_model_kwargs):
  pytest.importorskip("tensorflow")
  mod, params = tvmc.frontends.load_model(load_model_kwargs)
  # ... rest of the test here...
```

This also means you can fold `verify_load_model__keras` back into the test rather than having a helper function.